intermediate_source/char_rnn_classification_tutorial.py

```diff
@@ -364,19 +364,19 @@ def timeSince(since):
 
 start = time.time()
 
-for epoch in range(1, n_epochs + 1):
```

Comment:
`step` could get confusing considering we're using RNNs with multiple "time steps"... maybe `iter` and `n_iters`?